Web/Pages/Transactions/Transactions.razor

```diff
@@ -142,4 +145,28 @@
         searchString = text;
         table?.ReloadServerData();
     }
+
+    private string RowStyleFunc(TransactionSummary transactionSummary, int index)
+    {
+        var transactionType = transactionSummary.TransactionTypeAmounts.FirstOrDefault();
+        if (transactionType == null)
+        {
+            return string.Empty;
+            
+        }
+        var transactionTypeGroup = TransactionTypes.GetTransactionTypeGroup(transactionType.TransactionType);
+        return transactionTypeGroup == TransactionTypeGroup.Credit ? "background-color: rgba(255,0,0,0.2)" : "background-color: rgba(0,255,0,0.2)";
```

Comment:
We should use MudBlazor's theme colors here : https://mudblazor.com/features/colors